pkg/server/image_pull.go

```diff
@@ -403,3 +410,16 @@ func newTransport() *http.Transport {
 		ExpectContinueTimeout: 5 * time.Second,
 	}
 }
+
+// addEncryptedImagesPullOpts adds the necessary pull options to a list of
+// pull options if enabled.
+func (c *criService) encryptedImagesPullOpts() []containerd.RemoteOpt {
+	if c.config.ImageEncryption.KeyModel == criconfig.EncryptionKeyModelNode ||
+		c.config.ImageEncryption.KeyModel == "" {
```

Comment:
not sure why we would want "" unknown key model scenarios. Or is there a good reason for this?

Should probably put on a switch and make default an error.. but if so will need to change up to return error and fix calling code to fail pull request.

Reply:
The config struct convention of being embedded makes this the default behavior even if it is omitted. So the "" case for this routine would just be omitted. The discussion from earlier, was around, whether we should make this a default configuration

https://github.com/containerd/cri/pull/1340#discussion_r370482067